sources/packages/api/src/route-controllers/application/application.system.controller.ts

```diff
@@ -226,4 +227,20 @@ export class ApplicationSystemController extends BaseController {
 
     return { supportingData: supportingUser.supportingData };
   }
+
+  /**
+   * Archives one or more applications when 43 days
+   * have passed the end of the study period.
+   */
+  @ApiOkResponse({
+    description: "All application(s) were successfully archived.",
+  })
+  @Patch("archive")
+  async archiveApplication(): Promise<void> {
+    try {
+      await this.applicationService.archiveApplication();
```

Comment:
Please capture the user token and add the user id to the modifier column while updating the applications.